assets/js/app.js

```diff
@@ -205,7 +205,16 @@ $(function () {
 
     }).on('click', 'a[href="#refresh"]', function (event) {
         event.preventDefault();
-        location.reload();
+        $.ajax({
+            type: 'POST',
+            url:  '/v1/scrape',
+            processData: false,
+            data: JSON.stringify($('#configure').serializeArray()),
```

Comment:
You shouldn't need `processData` or `data`  Check out the definition of `/v1/scrape` server-side -- I don't believe it takes any data in the body.